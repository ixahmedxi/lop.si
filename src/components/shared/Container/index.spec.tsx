import { render } from 'test-utils'
import { Container } from '.'

describe('Container component', () => {
  it('matches snapshot', () => {
    const { container } = render(<Container />)
    expect(container).toMatchSnapshot()
  })
})
