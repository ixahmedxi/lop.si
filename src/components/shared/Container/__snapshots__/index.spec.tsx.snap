// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component matches snapshot 1`] = `
.emotion-0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  width: 100%;
  padding-left: 8px;
  padding-right: 8px;
  margin: 0 auto;
  max-width: 100%;
}

@media screen and (min-width:576px) {
  .emotion-0 {
    max-width: 540px;
  }
}

@media screen and (min-width:768px) {
  .emotion-0 {
    max-width: 720px;
  }
}

@media screen and (min-width:992px) {
  .emotion-0 {
    max-width: 960px;
  }
}

@media screen and (min-width:1200px) {
  .emotion-0 {
    max-width: 1140px;
  }
}

<div>
  <div
    class="emotion-0"
  />
</div>
`;
